media-sound/freac/freac-1.1~alpha20180913.recipe

```diff
@@ -0,0 +1,94 @@
+SUMMARY="Free and open source audio converter"
+DESCRIPTION="fre:ac is a free and open source audio converter. It comes with \
+an integrated tag editor and converts between various formats.
+
+Features include:
+
+- Support for MP3, MP4/AAC, Opus, FLAC and other formats
+- Integrated player and tag editor
+- Support for playlists, cue sheets and chapters
+- Fast, multi-threaded conversion engine
+- Ability to convert to multiple formats at once
+- Can verify lossless input and output files
+- Support for freedb/CDDB queries and submits
+- Support for using command line codecs
+- User interface available in 40+ languages
+- Optional command line interface"
+HOMEPAGE="https://freac.org/"
+COPYRIGHT="2001-2018 by Robert Kausch"
```

Comment:
```diff
-COPYRIGHT="2001-2018 by Robert Kausch"
+COPYRIGHT="2001-2018 Robert Kausch"
```
(We prefer to drop the "by" to be consistent with the other recipes.)
(Can you make this change in the other PRs?)